From b2a070f102e4b5af0157eda556f5dd144cee1160 Mon Sep 17 00:00:00 2001
From: Luigi Toscano <luigi.toscano@tiscali.it>
Date: Sun, 19 Oct 2014 16:52:06 +0200
Subject: [PATCH] Remove root-XML tag from application/x-basket-item

The tag needs a namespaceURI which does not make sense for its usage.
---
 file-integration/basket.xml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/file-integration/basket.xml b/file-integration/basket.xml
index 5714972..aaa9309 100644
--- a/file-integration/basket.xml
+++ b/file-integration/basket.xml
@@ -50,6 +50,5 @@
         <match type="string" offset="39" value="!DOCTYPE basket>"/>
       </match>
     </magic>
-    <root-XML localName="basket" />
   </mime-type>
 </mime-info>
-- 
2.1.1

